ftl/core/card-templates.ftl

```diff
@@ -21,6 +21,10 @@ card-templates-night-mode = Night Mode
 card-templates-add-mobile-class = Add Mobile Class
 card-templates-preview-settings = Options
 card-templates-invalid-template-number = Card template { $number } in notetype '{ $notetype }' has a problem.
+card-templates-identical-front = Its front side is identical to the one of card template { $number }.
+card-templates-missing-cloze = The 'cloze' filter must be used on both sides of a cloze template.
```

Comment:
```suggestion
card-templates-missing-cloze = Expected to find '{{cloze:Text}}' or similar on the front and back of the card template.

Reply:
{{FrontSide}} is valid on the back of a cloze template IIRC, and the current check doesn't handle it.

Reply:
I just checked and the implicit `{{cloze:text}}` in `{{FrontSide}}` isn't expanded, i.e. it doesn't reveal the answer when used on the back side.

Reply:
Ah, sorry, the brackets needed to be escaped